build/add-aad-test-environment.yml

```diff
@@ -20,7 +20,10 @@ steps:
 
       # Get admin token
       $username = "$(tenant-admin-user-name)"
-      $password_raw = '$(tenant-admin-user-password)'
+      $password_raw =
+      @"
+      $(tenant-admin-user-password)
```

Comment:
What was this for?

Reply:
it fix pipeline failure -- password contains single quote which breaks script. I mentioned that in team channel. :)